css/properties/backface-visibility.json

```diff
@@ -39,7 +39,13 @@
               {
                 "prefix": "-moz-",
                 "version_added": "10",
-                "version_removed": "preview"
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "layout.css.prefixes.transforms",
+                    "value_to_set": "true"
+                  }
+                ]
```

Comment:
This change looks wrong, because it states that `-moz-backface-visibility` was available since Firefox 10 only behind a preference. If that was indeed the case, then this support statement would need to be removed, because there is already a stable support for the unprefixed version.

But actually it looks to me as if that preference is enabled by default in release, so it doesn't seem to be right to add the flag here.

Reply:
Ah, the problem here is that the value is inverted from the usual. It is true when the thing we want to change in future is disabled. So this needs to be set `false`, and hopefully that fill fix this.

Reply:
Just to make sure we're on the same page:

- Before this PR, the support statement says: "The prefixed property has been supported since Firefox 10, but is removed in the current Firefox Nightly."
- In this PR, the support statement says: "The prefixed property has been supported only behind a preference since Firefox 10 - i.e. it was never enabled by default -, and to enable the feature, the `layout.css.prefixes.transforms` preference has to be set to `true`."

If the default value of the preference is equal to the value that enables the feature, then this feature is effectively enabled by default, and IMO the preference is irrelevant for BCD.

If the feature is disabled by default in Nightly, and can be re-enabled in Nightly with the preference, then what might work is to keep the "added in 10, removed in preview" statement, but add another statement saying "added in preview, flags = ...".

Reply:
Thanks @caugner !

Here is exactly what happened
- These features were added in FF10
- In FF120 they were removed in preview releases (by adding a preference, and setting that preference `false` in nightly).
- In FF126 they un-removed, by making that preference `true` always.

So I think from a BCD perspective this just undoes the original change in FF120, and the change I should make would be to remove the preview markup from FF120 so it looks like these were never on the path for removal.

Does that make sense?

Reply:
And now I see that is exactly what you suggested in https://github.com/mdn/browser-compat-data/pull/25300/files#r1867548035 and below. Thanks!